daemon/network.go

```diff
@@ -468,3 +468,35 @@ func (daemon *Daemon) deleteNetwork(networkID string, dynamic bool) error {
 func (daemon *Daemon) GetNetworks() []libnetwork.Network {
 	return daemon.getAllNetworks()
 }
+
+// clearAttachableNetworks removes the attachable networks
+// after disconnecting any connected container
+func (daemon *Daemon) clearAttachableNetworks() {
+	var networks []libnetwork.Network
+	for _, n := range daemon.GetNetworks() {
```

Comment:
Wondering, should we combine the two loops, and simply

```go
for _, n := range daemon.GetNetworks() {
    if !n.Info().Attachable() {
        continue
    }
```

Or is there a special reason for using the intermediate `networks` variable?